rasterio/_warp.pyx

```diff
@@ -567,6 +567,8 @@ def _calculate_default_transform(
             log.debug("Created transformer and warp output.")
         except CPLE_NotSupported as err:
             raise CRSError(err.errmsg)
+        except CPLE_AppDefined as err:
+            pass
```

Comment:
@perrygeo instead of only passing, let's call `log.debug()`. We should log when we catch an exception that isn't re-raised.